R/get_phylopic.R

```diff
@@ -2,25 +2,30 @@
 #'
 #' This retrieves a PhyloPic silhouette as a vectorized or rasterized object.
 #'
-#' @param uuid A PhyloPic image uuid.
-#' @param format Format of the image. To return a vectorized image, use
-#'   "vector". To return a rasterized image, use one of 512, 1024, or 1536.
-#'   Rasterized thumbnails can be returned by using 64, 128, or 192. Finally,
-#'   using "twitter" will return a rasterized image that includes the PhyloPic
-#'   logo and is formatted for posting on social media.
-#' @return If `format` is "vector", a [Picture][grImport2::Picture-class]
-#'   object is returned. Otherwise, a png array representing the rasterized
-#'   image is returned. Either way, the uuid and download url are included as
-#'   the "uuid" and "url" attributes, respectively.
+#' @param uuid \code{character}. A PhyloPic image uuid.
+#' @param format \code{character}. Format of the image. To return a vectorized
+#'   image, use "vector". To return a rasterized image, use "raster" and specify
+#'   a desired `height`. Rasterized square thumbnails can be returned by using
+#'   "64", "128", or "192". Finally, using "twitter" will return a rasterized
+#'   image that includes the PhyloPic logo and is formatted for posting on
+#'   social media.
+#' @param height \code{numeric}. If `format` is "raster", this is the desired
+#'   height of the raster image. This is ignored for all other `format`s.
+#' @return If `format` is "vector", a [Picture][grImport2::Picture-class] object
```

Comment:
Is it worth stating something about the width of the image in the details section? Something along the lines of it is automatically generated based on the height initial aspect ratio?